frontend/src/Menu/index.jsx

```diff
@@ -1,52 +1,51 @@
 import React from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { View, Dimensions } from 'react-native';
-import { TouchableImage } from '../common/components';
+import { StyleSheet } from 'react-native';
+import { CardView } from '../common/components';
 import { changeDisplayedComponent } from './actionCreator';
+import MenuItem from './MenuItem';
 import buildingMenuIcon from '../../assets/menu/Buildings.png';
 import troopsMenuIcon from '../../assets/menu/Troops.png';
 import battleMenuIcon from '../../assets/menu/Battle.png';
 import leaderBoardMenuIcon from '../../assets/menu/Leaderboard.png';
+import Colors from '../common/colors';
 
-const HEIGHT_RATIO = 0.25;
+const ICON_LIST = [
```

Comment:
why did you put this here from the component? will you need it somewhere else too? because you don't export it.
please don't change it back, it's okay for me, I'm just curious about this change!

Reply:
Icon list is not based on the props of the menu function, so it's better to mention it once instead of putting it inside of the function